src/index.js

```diff
@@ -0,0 +1,196 @@
+          }
+        </style>
+      </head>
+      <body>
+        <header>
+          <div class="collapse bg-dark" id="navbarHeader">
+            <div class="container">
+              <div class="row">
+                <div class="col-sm-8 col-md-7 py-4">
+                  <h4 class="text-white">${teamName.teamName}</h4>
+                  <p class="text-muted">
+                    Add some information about the album below, the author, or any
+                    other background context. Make it a few sentences long so folks
+                    can pick up some informative tidbits. Then, link them off to
+                    some social networking sites or contact information.
+                  </p>
+                </div>
+                <div class="col-sm-4 offset-md-1 py-4">
+                  <h4 class="text-white">Contact</h4>
+                  <ul class="list-unstyled">
+                    <li><a href="#" class="text-white">Follow on Twitter</a></li>
+                    <li><a href="#" class="text-white">Like on Facebook</a></li>
+                    <li><a href="#" class="text-white">Email me</a></li>
+                  </ul>
+                </div>
+              </div>
+            </div>
+          </div>
+          <div class="navbar navbar-dark bg-dark shadow-sm">
+            <div class="container">
+              <a href="#" class="navbar-brand d-flex align-items-center">
+                <i class="fas fa-users"></i>
+                <strong>${teamName.teamName}</strong>
+              </a>
+            </div>
+          </div>
+        </header>
+    
+        <main>
+          <section class="py-5 text-center container">
+            <div class="row py-lg-5">
+              <div class="mx-auto">
+                <h1 class="fw-light">🎉 Welcome to ${teamName.teamName}! 🎉</h1>
+                <p class="lead text-muted">
+                  Our team members are below, if you like the look of us and you
+                  think we can work together, why not fire over an email? 🚀
+                </p>
+              </div>
+            </div>
+          </section>
+  
+            <div class="album py-5 bg-light">
+            <div class="container">
+              <div class="d-flex flex-row flex-wrap">${cardSection}</div>
+            </div>
+          </div>
+        </main>
+    
+        <footer class="text-muted py-5 bg-dark">
+          <div class="container text-center text-white">
+            <p class="mb-1">
+              Team Profile Generator, made by
+              <a href="https://github.com/conorjkelly96">Conor Kelly</a>.
+            </p>
+          </div>
+        </footer>
+    
+        <script src="../assets/dist/js/bootstrap.bundle.min.js"></script>
+      </body>
+    </html>
+    `;
+};
+
+// initialize user interaction
+const init = async () => {
+  // prompt Main Questions
+  let inProgress = true;
+  const teamMembers = [];
+
+  const answers = await inquirer.prompt(teamBuilderQuestions);
```

Comment:
You can destructure `teamName` directly here.

Reply:
resolved